daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/Runtime.scala

```diff
@@ -67,11 +67,11 @@ import org.xml.sax.SAXException
 import edu.illinois.ncsa.daffodil.io.BitOrderChangeException
 import edu.illinois.ncsa.daffodil.infoset._
 import edu.illinois.ncsa.daffodil.processors.parsers.ParseError
-import edu.illinois.ncsa.daffodil.processors.parsers.Parser
 import edu.illinois.ncsa.daffodil.processors.parsers.PState
 import edu.illinois.ncsa.daffodil.exceptions.UnsuppressableException
 import edu.illinois.ncsa.daffodil.api.TunableLimitExceededError
 import edu.illinois.ncsa.daffodil.api.DaffodilTunables
+import edu.illinois.ncsa.daffodil.processors.parsers.Parser
```

Comment:
Can revert. This is just a artifact of changing Parser to ParserLike, then thinking better of it and switching back.